modules/icinga/files/usr/lib/nagios/plugins/check_directory_exists

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Desc : Plugin to verify a directory dxists
+
+PROGNAME="$(basename "$0")"
+
+if [ "$1" = "" ]
+then
+  echo -e " Use : $PROGNAME <directory_name> -- Ex : $PROGNAME /etc \n "
+  exit 3
+fi
+
+if [ -d "$1" ]
```

Comment:
Just to check, this is fine that the directory is a symlink? it worked on a quick test of my own so seems good.

Reply:
Yeah, I remember checking this specific case, and yep, it works if the symlink is working, and fails (as intended) if the symlink is broken.